Library/Formula/fabric.rb

```diff
@@ -0,0 +1,49 @@
+require "formula"
+
```

Comment:
@robsonpeixoto I’m not a maintainer, and this is in `brew audit` but you have to add the `--strict` option ;)